.eslintrc.json

```diff
@@ -1,24 +1,27 @@
 {
-  "env": {
-    "node": true,
-    "commonjs": true,
-    "es2017": true,
-    "mocha": true
-  },
-  "ignorePatterns": ["**/*.mjs", "**/*.js"],
+  "ignorePatterns": ["build", "coverage"],
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended-type-checked",
     "plugin:@typescript-eslint/stylistic-type-checked",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
     "plugin:prettier/recommended"
   ],
-  "plugins": ["@typescript-eslint", "import"],
+  "settings": {
+    "import/resolver": {
+      "typescript": true
```

Comment:
This activates the resolver added to the package, needed to properly resolve TypeScript imports.